http/src/main/java/io/opentelemetry/example/http/HttpServer.java

```diff
@@ -13,43 +13,24 @@
 import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.api.trace.SpanKind;
-import io.opentelemetry.api.trace.Tracer;
-import io.opentelemetry.context.Context;
-import io.opentelemetry.context.Scope;
-import io.opentelemetry.context.propagation.TextMapGetter;
-import io.opentelemetry.context.propagation.TextMapPropagator;
+import io.opentelemetry.extension.incubator.trace.ExtendedTracer;
 import java.io.IOException;
 import java.io.OutputStream;
 import java.net.InetSocketAddress;
 import java.nio.charset.Charset;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 public final class HttpServer {
   // It's important to initialize your OpenTelemetry SDK as early in your application's lifecycle as
   // possible.
   private static final OpenTelemetry openTelemetry = ExampleConfiguration.initOpenTelemetry();
-  private static final Tracer tracer =
-      openTelemetry.getTracer("io.opentelemetry.example.http.HttpServer");
+  private static final ExtendedTracer tracer =
+      ExtendedTracer.create(openTelemetry.getTracer("io.opentelemetry.example.http.HttpServer"));
```

Comment:
Its arguably a bit strange to reference experimental APIs as part of canonical examples, but I think its fine if we're serious about incubating these to eventually incorporate into Tracer.